autocomplete/complete.go

```diff
@@ -67,7 +68,17 @@ func getPotentialTarget(prefix string) ([]string, error) {
 	}
 	dirPath := splits[0]
 
-	targets, err := earthfile2llb.GetTargets(path.Join(dirPath, "Earthfile"))
+	user, err := user.Current()
```

Comment:
Might be worth using a different variable name for `user` to avoid overlap with the package name `user`?

Reply:
ha, that's a really good point. I didn't notice I did that :)